gslab_scons/provenance.py

```diff
@@ -100,11 +101,13 @@ def scan(start_path, include_details, include_checksum, detail_limit, file_detai
     return total_size, num_files, last_mtime, file_details, dirs
 
 
-def write_heading(start_path, provenance_path):
+def write_heading(start_path, provenance_path, github_release = None):
     '''
     Write standard heading for provenance: what and for where it is. 
     '''
     out = '*** GSLab directory provenance ***\ndirectory: %s\n' % os.path.abspath(start_path) 
+    if github_release != None:
+        out += '\n*** GitHub release: %s \n' % github_release
```

Comment:
Breaks pattern since the release information appears on the same line as the section header. Consider breaking the text over two lines.

Reply:
It's not really a section header as writing `directory: <stuff>`

Reply:
If it's not a section heading, then it shouldn't be formatted as a section heading (`***`). Maybe just format it like the `directory` entry?

Reply:
Per discussion, we're formatting the `directory` entry like `Github release`.